### PR TITLE
DownloadImportFailureCsv File added BOM_UTF8 and unnecessary remove BOM_UTF8

### PR DESCRIPTION
**Pull request by corean**
## Description

It removed the part that added BOM_UTF8 to subsequent CSV files, and consequently removed BOM_UTF8.

As long as there is BOM_UTF8 only at the beginning (header) of the final merged CSV file, the file opens without any encoding issues.
[6e0ba25](https://github.com/filamentphp/filament/commit/6e0ba25bf04c3796cb865e1b42a5c67e248cbe43)

## Visual changes

none

## Functional changes

- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

### CODE REVIEW

**Comment by danharrin** [comment]
I don't understand, sorry. Can you please check #12197?

**Comment by corean** [comment]
> I don't understand, sorry. Can you please check #12197?

updated description

File: packages/actions/src/Imports/Http/Controllers/DownloadImportFailureCsv.php
Comment (danharrin) [inline packages/actions/src/Imports/Http/Controllers/DownloadImportFailureCsv.php:1]: Do you need to use `'Content-Type' => 'text/csv; charset=UTF-8',` like you did in the other PR, or does that not matter?

File: packages/actions/src/Exports/Downloaders/CsvDownloader.php
Comment (danharrin) [inline packages/actions/src/Exports/Downloaders/CsvDownloader.php:1]: same here?

File: packages/actions/src/Imports/Http/Controllers/DownloadImportFailureCsv.php
Comment (corean) [reply packages/actions/src/Imports/Http/Controllers/DownloadImportFailureCsv.php:1]: no matter. 
Updated the code and committed the changes

**Comment by danharrin** [comment]
Thanks

**Pull request merged**